app/lib/service/security_advisories/backend.dart

```diff
@@ -93,20 +95,73 @@ class SecurityAdvisoryBackend {
             osv.published != null ? DateTime.parse(osv.published!) : modified
         ..syncTime = syncTime;
 
-      tx.queueMutations(
-        // This is an upsert
-        inserts: [newAdvisory],
-      );
+      final packages = await _lookupAffectedPackages(newAdvisory, tx);
+      if (packages.length > 50) {
```

Comment:
Just do `advisory.affectedPackages.length` before calling `_lookupAffectedPackages(newAdvisory, tx);`

otherwise, if there's 500 affected packages you'll try to load them all, and then conclude that you don't want to do anything.

Reply:
done